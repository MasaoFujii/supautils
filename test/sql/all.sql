create role not_yet_created;
