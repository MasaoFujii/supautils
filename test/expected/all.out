create role not_yet_created;
ERROR:  The "not_yet_created" role is reserved, only superusers can create it.
